en/guide/overview.md

```diff
@@ -1,13 +1,56 @@
 ## Overview
 
-The diagram below shows the main components of a system that is running the *Camera Streaming Daemon* on a Linux companion computer.
+The diagram below shows the main components of a "typical" system that is running the *Camera Streaming Daemon* on a Linux companion computer.
+CSD connects to multiple cameras that are attached to the linux computer. 
+It provides access to them via the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html) and RTSP video streams. It can also advertise available RTSP streams using Avahi. 
 
 ![Camera streaming daemon overview](../../assets/camera_streaming_daemon_overview.png)
 
-The server:
+In the diagram above the GCS requests for camera actions are forwarded by PX4 - CSD can also take MAVLink requests directly, and may do so in different configurations.
 
-* Scans and attaches to all compatible cameras. Cameras that support the [Video4Linux (V4L2) API](https://linuxtv.org/downloads/v4l-dvb-apis/uapi/v4l/v4l2.html) work out of the box (the server can be extended to support other camera protocols/APIs).
-* Advertises and shares RTSP video streams for all connected cameras. These can be consumed by the GCS and other video player services.
-* Captures and stores/shares video and still images for all connected cameras via the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html). GCS requests for camera actions are forwarded by PX4. 
 
-> **Note** The *Camera Streaming Daemon* also integrates with Gazebo, providing a simulated camera backend. In this case the diagram/configuration can be slightly different as the GCS can communicate directly with the server (there is no need for messages to be forwarded by PX4).
\ No newline at end of file
+## Key Features
+
+CSD supports the following key features:
+
+* Automatically attaches [compatible cameras](#supported_cameras) connected to the Linux computer.
+* RTSP video streaming from *all* connected cameras (for consumption by GCS or other video players).
+* RTSP video stream advertising/discovery using Avahi.
+* [MAVLink Camera Protocol](#mavlink_support) support for up to 5 cameras, enabling video and image capture and storage, and querying/setting camera options.
+* Gazebo simulated camera backend (so you can view video streams from within a simulated environment)!
+* Configurable back-end that can be extended to interface with new types of cameras and new front-end protocols.
+
+
+## Supported Cameras {#supported_cameras}
+
+Cameras that support the [Video4Linux (V4L2) API](https://linuxtv.org/downloads/v4l-dvb-apis/uapi/v4l/v4l2.html) should work out of the box. 
+The server can be [extended](../guide/extending_csd.md) to support other back-end camera protocols/APIs.
+
+Some readily-available cameras that have been used in testing include: 
+* Logitech C270 HD Webcam
+* Sony PlayStation Eye Camera
+* Intel RealSense 3D Camera
+
+## Configuration Options {#configuration}

```

Comment:
FYI This is supposed to provide entry point into the idea that the system is configurable at different points.